src/environments/.env.ts

```
@@ -1,3 +1,3 @@
 export default {
-  'npm_package_version': '0.0.0'
+  'npm_package_version': '1.1.3'
```

dotenv-linter (3.3.0)

[warning] [IncorrectDelimiter] The 'npm_package_version': '1.1.3' key has incorrect delimiter

(IncorrectDelimiter)